src/runtime.cc

```diff
@@ -60,16 +62,16 @@ namespace detail {
 #endif
 	}
 
-	std::string get_sycl_version() {
-		std::stringstream ss;
+	const char* get_sycl_version() {
 #if defined(__COMPUTECPP__)
-		ss << "ComputeCpp " << COMPUTECPP_VERSION_MAJOR << "." << COMPUTECPP_VERSION_MINOR << "." << COMPUTECPP_VERSION_PATCH;
-		return ss.str();
+		return "ComputeCpp " CELERITY_STRINGIFY(COMPUTECPP_VERSION_MAJOR) "." CELERITY_STRINGIFY(COMPUTECPP_VERSION_MINOR) //
```

Comment:
I mean.... We do have access `fmt::format` as well ;-).

Reply:
But this is compile time! Please consider the highly significant performance gains in building the version string.

Reply:
I honestly forgot about `fmt` when doing this :)